template/private-npm-package/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "<%- fullModuleName %>",
+  "description": "<%- description %>",
+  "version": "0.0.0-semantically-released",
+  "main": "lib-commonjs",
+  "module": "lib-es2015",
+  "typings": "lib-types",
+  "files": [
+    "lib"
+  ],
+  "sideEffects": false,
+  "license": "UNLICENSED",
+  "scripts": {
+    "build": "skuba build",
+    "commit": "skuba commit",
+    "format": "skuba format",
+    "format:check": "skuba format check",
+    "lint": "skuba lint",
+    "prerelease": "skuba build",
+    "release": "skuba release",
+    "test": "echo \"No tests for now\""
+  },
+  "repository": {
+    "type": "git"
```

Comment:
Does this get rendered with the repository URL after templating? I think that _may_ be required to release the package, but you'll have to double check.